.editorconfig

```diff
@@ -0,0 +1,18 @@
+[*]
```

Comment:
Is this some from specific editor?

Reply:
Thanks @ianwallen some of my co-workers have explained the duplication is by design presently. Although it may be possible to improve the level of reuse @josegar74 ?